Package.swift

```diff
@@ -24,7 +24,7 @@ let package = Package(
         .package(url: "https://github.com/maplibre/maplibre-gl-native-distribution.git", from: "6.4.0"),
         .package(url: "https://github.com/stadiamaps/maplibre-swift-macros.git", from: "0.0.3"),
         // Testing
-        .package(url: "https://github.com/Kolos65/Mockable.git", exact: "0.0.3"),
+        .package(url: "https://github.com/Kolos65/Mockable.git", branch: "main"),
```

Comment:
this will be replaced by the other PR once that one is merged.

Reply:
Merged. You may still get a warning like this, but it's harmless. Eventually the upstream will release (they merged my PR correcting the issue).

```
Showing Recent Messages
'maplibre-swift-macros' dependency on 'https://github.com/swiftlang/swift-syntax.git' conflicts with dependency on 'https://github.com/apple/swift-syntax.git' which has the same identity 'swift-syntax'. this will be escalated to an error in future versions of SwiftPM.
```